src/ddf-rules/data-package-rules/datapoint-without-indicator.ts

```diff
@@ -1,18 +1,17 @@
-import {compact, difference, flattenDeep, map, isArray, isEmpty} from 'lodash';
-import {DdfDataSet} from '../../ddf-definitions/ddf-data-set';
-import {Issue} from '../issue';
-import {DATA_POINT_WITHOUT_INDICATOR} from '../../ddf-rules/registry';
+import { compact, difference, flattenDeep, map, isArray, isEmpty } from 'lodash';
+import { DdfDataSet } from '../../ddf-definitions/ddf-data-set';
+import { Issue } from '../issue';
+import { DATA_POINT_WITHOUT_INDICATOR } from '../../ddf-rules/registry';
 
-const getIssues = (ddfDataSet: DdfDataSet): Array<Issue> => {
-  const result: Array<any> = ddfDataSet.ddfRoot.directoryDescriptors.map(directoryDescriptor =>
-    directoryDescriptor.dataPackage.dataPackage.resources
-      .filter(resource => isArray(resource.schema.primaryKey))
-      .filter(resource => isEmpty(difference(map(resource.schema.fields, 'name'), resource.schema.primaryKey)))
-      .map(resource => new Issue(DATA_POINT_WITHOUT_INDICATOR).setPath(resource.path).setData(resource)));
+const getIssues = (ddfDataSet: DdfDataSet): Issue[] => {
+  const result: any[] = ddfDataSet.ddfRoot.dataPackage.dataPackage.resources
```

Comment:
https://ru.wikipedia.org/wiki/%D0%97%D0%B0%D0%BA%D0%BE%D0%BD_%D0%94%D0%B5%D0%BC%D0%B5%D1%82%D1%80%D1%8B
Avoid train-wrecks in you code